### ISSUE TITLE
Resizing window resets translation textarea heights

### ISSUE THREAD

**Issue by sushain97** [goavki/apertium-html-tools#151]
I really want it to retain my heights. Resetting them to whatever the app wants is silly and I'm not sure when this bad behavior creeped in.

@share-with-me perhaps this can be fixed with #142?

**Comment by share-with-me** [comment]
That means, when the window is resized, `synchronizeTextareaHeights` should not be called? That is, the heights of the textareas should stay the same even on resizing?

**Comment by sushain97** [comment]
I don't know exactly what `synchronizeTextareaHeights` does. But that sounds like a reasonable conclusion. Can you figure out why it was made to do that? `git blame` might be helpful.

**Comment by share-with-me** [comment]
On it! Meanwhile could you please review the comments on #142 ?! Thanks!

**Comment by share-with-me** [comment]
> I really want it to retain my heights.

Um, do you mean in the event that I manually resize it using the dragger at the bottom right? Because currently, when I paste in a lot of content, the `textarea` dynamically resizes itself and retains its height even on `window` resize?!

**Comment by sushain97** [comment]
> Um, do you mean in the event that I manually resize it using the dragger at the bottom right? 

Yes.

> On it! Meanwhile could you please review the comments on #142 ?! Thanks!

I'll take a look when I'm free. I think this issue and that one are connected/can be solved together.

**Comment by share-with-me** [comment]
> I'll take a look when I'm free. I think this issue and that one are connected/can be solved together.

Thanks :D

**Comment by sushain97** [comment]
Assigning to myself since #152 is blocked by me figuring out sane requirements.

**Comment by Androbin** [comment]
Any updates on this yet?

**Comment by sushain97** [comment]
No. If you would like to figure out the prons and cons of each approach, feel free!

**Comment by Androbin** [comment]
Well, for one thing, I could resolve #152.

**Comment by sushain97** [comment]
I don't know what it means to resolve a pull request.

The blocking item here is that the correct behavior for each scenario needs to be articulated. See my comments here and on #132 and #152 for different problems I've brought up.

**Comment by Androbin** [comment]
> I don't know what it means to resolve a pull request.

I meant resolving the merge conflicts and addressing the issues brought up.

**Comment by sushain97** [comment]
I would rather nothing happen until a solid plan is formed.
